src/redux/fetch/booksFetcher.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const base = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+const url = `${base}/apps/Soeu3oY0gBxHutNA3nzF/books`;
```

Comment:
- [Optional] Not really good practice to push confidential info to the repo. This info could be hidden and ignored in an env file.